docs/examples/color_space_conversion.ipynb

```diff
@@ -0,0 +1,170 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Color Space Conversion\n",
+    "\n",
+    "This example showcases different color space conversion operations available in DALI."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import nvidia.dali.ops as ops\n",
+    "import nvidia.dali.types as types\n",
+    "from nvidia.dali.pipeline import Pipeline\n",
+    "\n",
+    "image_dir = \"images\"\n",
+    "batch_size = 8\n",
```

Comment:
In L32 it is set to 32, so it can be removed here.

Reply:
done